```
======================================================================
FAIL: test_iterate_repeat_records (corehq.apps.repeaters.tests.test_dbaccessors.TestRepeatRecordDBAccessors)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/repeaters/tests/test_dbaccessors.py", line 100, in test_iterate_repeat_records
    self.assertEqual(len(records), 3)  # Should grab all but the succeeded one
AssertionError: 0 != 3
```

--- a/corehq/apps/repeaters/tests/test_dbaccessors.py
+++ b/corehq/apps/repeaters/tests/test_dbaccessors.py
@@ -27,25 +27,25 @@ class TestRepeatRecordDBAccessors(TestCase):
             domain=cls.domain,
             failure_reason='Some python error',
             repeater_id=cls.repeater_id,
-            next_event=before,
+            next_check=before,
         )
         success = RepeatRecord(
             domain=cls.domain,
             succeeded=True,
             repeater_id=cls.repeater_id,
-            next_event=before,
+            next_check=before,
         )
         pending = RepeatRecord(
             domain=cls.domain,
             succeeded=False,
             repeater_id=cls.repeater_id,
-            next_event=before,
+            next_check=before,
         )
         other_id = RepeatRecord(
             domain=cls.domain,
             succeeded=False,
             repeater_id=cls.other_id,
-            next_event=before,
+            next_check=before,
         )
 
         cls.records = [

